server/plugins/client/bundle-handler-factory.js

```diff
@@ -0,0 +1,54 @@
+module.exports.createHandler = createHandler
```

Comment:
`index.js` is because it’s a hapi plugin, which has certain requirement: http://hapijs.com/tutorials/plugins